### ISSUE TITLE
Search method in google chat spaces

### ISSUE THREAD

**Issue by SantoshKumarRavi** [googleapis/google-api-nodejs-client#3518]
Thanks for stopping by to let us know something could be better!

**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.

 **Is your feature request related to a problem? Please describe.**
I tried to use search method of google chat space which is already developed by google but it is not available in this npm package. There are get, list, create methods but search is missing.
 **Describe the solution you'd like**
Basically we can develop this feature like another methods in google apis
 **Describe alternatives you've considered**
For a temporary solution, i can use the Raw url which is provided by google.
 **Additional context**
https://developers.google.com/workspace/chat/api/reference/rest/v1/spaces/search

I checked in latest version too. search method is not available. Let me know if I am missing any implementation for search in google chat space apis.

**Comment by sofisl** [comment]
Hi there! If it is not available it means that the client libraries have not yet released it - apologies. I *think* it's been released (if [this](https://github.com/googleapis/google-api-nodejs-client/blob/f4848c7295f887ae311c93967ee4723e336f263c/src/apis/chat/v1.ts#L3558) is the same one you are looking for). Otherwise it may just not yet be available.
